### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    default = "5g"
+
+[resources]
+        [resources.sources.main]
+        arm64.url = "https://git.pleroma.social/pleroma/pleroma/-/jobs/234437/artifacts/download?file_type=archive"
+        arm64.sha256 = "2e2622a7a50284573fbb3dc66f0510c03d19403ab1e23d6b5dce7bddc11fca95"
+        amd64.url = "https://git.pleroma.social/pleroma/pleroma/-/jobs/234433/artifacts/download?file_type=archive"
+        amd64.sha256 = "8ef0bea62671d39e60f9e08d13109a4c332c552a1f855184063353987d46c84a"
+        armhf.url = "https://git.pleroma.social/pleroma/pleroma/-/jobs/234449/artifacts/download?file_type=archive"
+        armhf.sha256 = "abab2ad69704b999dbe1b995ce12aabc8069d9620ca332ecc621b20826d6db97"
```
Comment (seanking2919) [inline manifest.toml:75]: Not 100% sure how to test out configuring the autoupdate strategy, but if possible, can YunoHost use Pleroma's GitLab repository?

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**